api/src/main/java/com/redhat/ipaas/api/v1/model/ConnectorGroup.java

```diff
@@ -21,30 +21,30 @@
 import java.io.Serializable;
 
 /**
- * ComponentGroups are labels in Camel.
+ * ConnectorGroups are labels in Camel.
```

Comment:
If they are labels in camels, why not calling it labels here, too ?

IMO a "group" is something that contains something, so an explicit way to group stuff. A "label" is something attached to an object, and the group of obects attached with the same label build an implicit grouping. 

You change a group by adding / removing references to the members (where maybe the member has even a reference to its parent groups), labels are much more decoupled.

I don't know the semantics which are supposed to be used here, but a "Group" and "Label" are quite different concepts which cannot be easily mapped frictionless in the backend.